scrapers/MindGeekAPI.py

```diff
@@ -0,0 +1,249 @@
+import difflib
+import json
+import os
+import re
+import sys
+from configparser import ConfigParser, NoSectionError
+from datetime import datetime
+from urllib.parse import urlparse
+
+import requests
+
+# Not necessary but why not ?
+user_agent = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:79.0) Gecko/20100101 Firefox/79.0'
+
+# Set Variable
+set_ratio = 0.8
+set_fileurl = "MindGeekAPI.ini"
+
+
+def scraping_url(url):
+    id, instance_token = get_info(url)
+    headers = {
+        'Instance': instance_token,
+        'User-Agent': user_agent
+    }
+    return id, headers
+
+
+def print_exit(q):
+    print(q, file=sys.stderr)
+    exit(1)
+
+
+def get_info(url):
+    todaystr = datetime.today().strftime('%Y-%m-%d')
+    today = datetime.strptime(todaystr, '%Y-%m-%d')
+    token, id = check_config(url, today)
+    if not token:
+        print("No instance token found, sending request...", file=sys.stderr)
+        try:
+            r = requests.get(url, timeout=(3, 5))
+        except requests.Timeout:
+            print_exit("Request Timeout")
```

Comment:
here you are only excepting for timeout errors
you should another catchall `except:` or just one error print for all errors
You can try this by adding a bogus url eg `not_a_url` to a scene when editing and try scraping

Reply:
Catch-all excepts should not be used. Instead use the parent exception thrown by requests: `requests.exceptions.RequestException`